### DISCUSSION TITLE
How to Change the Favicon

### DISCUSSION THREAD

**Discussion by realCmdData** [onweru/compose#51] [Q&A] [answered]
I've tried to add a favicon.svg file to the content folder and the root folder but the website still shows the writing hand icon

**Comment by onweru** [comment, marked as answer]
@realCmdData,

Inside your `static` directory create a `favicons` directory with [these files](https://github.com/onweru/compose/tree/master/static/favicons). Use [this tool](https://realfavicongenerator.net)  to generate the files

**Reply by realCmdData** [reply]
Thank you very much!
